Library/Homebrew/cli/parser.rb

```diff
@@ -123,12 +135,18 @@ def initialize(&block)
 
         @args = Homebrew::CLI::Args.new
 
-        # Filter out Sorbet runtime type checking method calls.
-        cmd_location = T.must(caller_locations).select do |location|
-          T.must(location.path).exclude?("/gems/sorbet-runtime-")
-        end.fetch(1)
-        @command_name = T.must(cmd_location.label).chomp("_args").tr("_", "-")
-        @is_dev_cmd = T.must(cmd_location.absolute_path).start_with?(Commands::HOMEBREW_DEV_CMD_PATH)
+        if cmd
+          @command_name = cmd.command_name
+          @is_dev_cmd = cmd.name&.start_with?("Homebrew::DevCmd")
+        else
+          # FIXME: remove once commands are all subclasses of `AbstractCommand`:
```

Comment:
This is an especially nice change, IMO. No more inspecting the call stack to figure out the command!